build/sonic_build_script.sh

```diff
@@ -251,19 +272,28 @@ clone_ws()
     git clone -b $BRANCH $GIT_HUB_URL
     check_error $? "git-clone"
     cd sonic-buildimage
-    if [ -v BRANCH_COMMIT ]; then
+    echo "docker system prune -a --volumes -f" > build_cmd.txt
```

Comment:
docker system prune is executed optionally with CLEAN_DOCKERS flag. So it will be better to move this echo where it is executed.